usr/src/uts/common/fs/zfs/lua/lstrlib.c

```diff
@@ -34,7 +34,6 @@
  */
 #define tolower(C)	(((C) >= 'A' && (C) <= 'Z') ? (C) - 'A' + 'a' : (C))
 #define toupper(C)      (((C) >= 'a' && (C) <= 'z') ? (C) - 'a' + 'A': (C))
-#define iscntrl(C)	((((C) >= 0) && ((C) <= 0x1f)) || ((C) == 0x7f))
```

Comment:
I have used kmdb libstand as reference for it.
to me - will be more better to move functions per file and add it to shared place - and add to build where we need this function, but it is not for this update